### DISCUSSION TITLE
Add racefans.net race scores / x1z.net most clicked

### DISCUSSION THREAD

**Discussion by JJWatMyself** [f1multiviewer/issue-tracker#217] [Feature requests]
x1z.net is a well worth visit to find the best races to watch.  I would love to see this racefans.net vote info / ranking being incorporated into MultiViewer.  The click tracking code (redir.php) on x1z.net is also pretty cool for the most clicked content.  A great way to help people find content 

https://x1z.net/racefans.php

**Comment by SnowLeopard71** [comment]
The Racefans' Top 100 I have on x1z.net is outdated -- it only covers up to the 2018 season.   

[Racefans Top 100](https://www.racefans.net/rate-the-race/f1-fanatic-top-100/)
